src/llama-model-loader.h

```diff
@@ -160,3 +165,7 @@ struct llama_model_loader {
 
     void print_info() const;
 };
+
+// return a list of splits for a given path
+// for example, given "<name>-00002-of-00004.gguf", returns list of all 4 splits
+std::vector<std::string> llama_get_list_splits(const std::string & path, const int n_split);
```

Comment:
This can be `static` function in the source file only - no need to add it in the header.

There is also an existing a `llama_split_` prefix which seems suitable to use for this function: `llama_split_get_list()`

Reply:
Ah yeah, I wanted to use this in `llama.cpp` but decided not to do that in the end. Forgot to delete it in the header file.

It should be fixed with https://github.com/ggerganov/llama.cpp/pull/11255/commits/49822bab15a584343ff3f01333beb378557215a6